set.mm

```diff
@@ -40133,22 +40187,55 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
   $}
 
   ${
-    $d x A $.
+    $d x y z $.  $d x A $.
     $( A class is equal to the empty set if and only if it has no elements.
-       Theorem 2 of [Suppes] p. 22.  (Contributed by NM, 29-Aug-1993.) $)
+       Theorem 2 of [Suppes] p. 22.  (Contributed by NM, 29-Aug-1993.)  Avoid
+       ~ ax-11 , ~ ax-12 .  (Revised by Gino Giotto, 28-Jun-2024.) $)
     eq0 $p |- ( A = (/) <-> A. x -. x e. A ) $=
-      ( nfcv eq0f ) ABABCD $.
+      ( vy vz c0 wceq cvv cdif cv wcel wn wal eqeq2i wb wfal bibi2i biidd bitri
+      cab weq df-nul wa df-dif dfcleq pm3.24 bifal abbii eleq2i vex ax-mp nbfal
+      elabgw bicomi albii ) BEFBGGHZFZAIZBJZKZALZEUOBUAMUPBCIGJZVAKUBZCSZFZUTUO
+      VCBCGGUCMVDURUQVCJZNZALUTABVCUDVFUSAVFURUQOCSZJZNZUSVEVHURVCVGUQVBOCVBVAU
+      EUFUGUHPVIURONZUSVHOURUQGJVHONAUIOOOCDUQGCDTOQDATOQULUJPUSVJURUKUMRRUNRRR
+      $.
+    $( $j usage 'eq0' avoids 'ax-10' 'ax-11' 'ax-12'; $)
```

Comment:
could be shortened by using the eq0f proving, but replacing steps 1 to 3 with ~ dfcleq
![image](https://github.com/metamath/set.mm/assets/58114641/a7f58edc-09b5-4b28-a30e-d71d8319ea83)

Reply:
Done